docs/source/tracking.rst

```diff
@@ -76,6 +76,104 @@ There are different kinds of remote tracking URIs:
   `[Azure] <http://docs.microsoft.com/azure/databricks/applications/mlflow/access-hosted-tracking-server>`_, or :ref:`the quickstart <quickstart_logging_to_remote_server>` to
   easily get started with hosted MLflow on Databricks Community Edition.
 
+How Runs and Artifacts are Recorded
+===================================
+As mentioned above, MLflow runs can be recorded to local files, to a SQLAlchemy compatible database, or remotely to a tracking server.
+For storing runs and artifacts, MLflow uses two components for `storage <https://mlflow.org/docs/latest/tracking.html#storage>`_: backend store and artifact store. While the backend store persists
+MLflow entities (runs, parameters, metrics, tags, notes, metadata, etc), the artifact store persists artifacts
+(files, models, images, in-memory objects, or model summary, etc).
+
+Which `backend store <https://mlflow.org/docs/latest/tracking.html#backend-stores>`_ or `artifact store <https://mlflow.org/docs/latest/tracking.html#artifact-stores>`_ is used by the MLFlow client or tracking server
+depends on four different scenarios.
+
+Scenario 1: MLflow on the localhost
```

Comment:
Nit: let's drop the `the` from `the localhost` across these scenario titles, it's cleaner.

![image](https://user-images.githubusercontent.com/39497902/105538288-08648a80-5ca8-11eb-8e45-2b00ef8c5473.png)

```suggestion
Scenario 1: MLflow on localhost
```

Reply:
lol!

Reply:
done!